Lesson_3_Introduction_to_Pandas_I/Lesson_3.ipynb

```diff
@@ -6,9 +6,11 @@
    "source": [
```

Comment:
<p>This looks okay on the jupyter notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>